dev-python/python-xmp-toolkit/python-xmp-toolkit-2.0.1-r2.ebuild

```diff
@@ -4,7 +4,7 @@
 EAPI=8
 
 DISTUTILS_USE_PEP517=setuptools
-PYTHON_COMPAT=( python3_{9..11} )
+PYTHON_COMPAT=( python3_{9..11} pypy3 )
```

Comment:
This one goes before pikepdf. Also, I get a bunch of test failures:
[dev-python:python-xmp-toolkit-2.0.1-r2:20230219-184940.log](https://github.com/gentoo/gentoo/files/10777756/dev-python.python-xmp-toolkit-2.0.1-r2.20230219-184940.log)

Reply:
This is known breakage, I think: https://bugs.gentoo.org/883939. Upstream seems dead too unfortunately.

Reply:
Thanks for testing. I wrote earlier that I had issues testing it. I'll look further.

Reply:
The python-xmp-toolkit failures have nothing to do with pypy3 ; with my fork in https://github.com/exmakhina/python-xmp-toolkit/ the tests pass with pypy and python ; without they fail with both implementations.

Reply:
(cf. https://bugs.gentoo.org/883939 ; I could add a temporary patch for python-xmp-toolkit in this merge request if desired)

Reply:
What's the patch?

Reply:
Merged in #29826.